src/alto-ui/Form/TextField/TextField.scss

```diff
@@ -23,13 +23,13 @@
     color: $coolgrey-20;
   }
 
-  &:hover:not(.textfield--readOnly):not([disabled]) {
+  &:hover:not(.textfield--readOnly):not([disabled]):not(.DatagridCellInput) {
     border-color: $blue;
     box-shadow: 0 0 0 1px $blue;
   }
 
-  &:focus:not(.textfield--readOnly),
-  &:focus:hover:not(.textfield--readOnly) {
+  &:focus:not(.textfield--readOnly):not(.DatagridCellInput),
+  &:focus:hover:not(.textfield--readOnly):not(.DatagridCellInput) {
```

Comment:
Cant validate that. We cant make ref to external component. We need to find a better way to solve it.

Reply:
I removed these refs